pyproject.toml

```diff
@@ -7,7 +7,7 @@ requires-python = ">=3.9"
 license = "MIT"
 authors = [{ name = "OpenAI", email = "support@openai.com" }]
 dependencies = [
-    "openai>=1.66.5",
+    "openai>=1.68.2",
```

Comment:
I think this should not be part of the PR, as it has nothing to do with the added example.

Reply:
In my local testing, it was neccessary, otherwise it wouldn't work with `1.66.5`.

Happy to change it though if that gets this through.